live/public/crossdomain.xml

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
On n'a pu besoin de ce fichier ? Why ?

Reply:
Non. https://www.emberjs.com/blog/2018/01/01/ember-2-18-released.html#toc_code-crossdomain-xml-code-removed-for-new-applications